data/json/monsters.json

```diff
@@ -1651,13 +1651,13 @@
     "id": "mon_dog_skeleton",
     "type": "MONSTER",
     "name": "skeletal dog",
-    "description": "This canines see-through thin, black veined and scab riddled skin, is tightly compressed around its skeletal frame.",
+    "description": "This canine skeletal frame with bulging bones, woven with a net of sinew growing out from conglutinated carapace of fused together ribs, resembles an overactive turtle.",
     "default_faction": "zombie",
     "species": [ "ZOMBIE" ],
     "diff": 12,
     "size": "SMALL",
     "hp": 12,
-    "speed": 120,
+    "speed": 100,
```

Comment:
Why the decrease?

Reply:
As stated in  #23463 the overgrown bone carapace and reduced muscle mass would reduce speed.